step_definitions/adminCrossSchoolPermission-steps.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+const loginPage = require('../page-objects/pages/loginPage');
+const startPage = require('../page-objects/pages/startPage');
+const loginData = require('../shared-objects/loginData');
+const firstLogin = require('../shared_steps/firstLogin.js');
+const elementHelpers = require('../runtime/helpers/elementHelpers.js');
+const administration = require("../page-objects/administration.js")
+
+
+
+
+
+Given(/^an admin is logged in$/, async function () {
+    await elementHelpers.loadPage(loginData.url, 10);
+    await startPage.clickLoginBtn();
+    await loginPage.performLogin('admin@schul-cloud.org', 'Schulcloud1!');
+    await firstLogin.firstLoginAdmin();
+});
+
+Given(/^an admin are able to request information about students from own school$/, async function () {
+    await administration.getStudentsFromSameSchoolAndVerify()
+});
+
+When(
+    /^admin tries to request information about students from other school$/,
+    function () {
+
```

Comment:
which function?

Reply:
Good question!
Does every integration test need a GIVEN - WHEN - THEN? Because I don't know what to put in the WHEN without calling the same functions twice. I'd be happy to remove the WHEN but then the tests reads out 'GIVEN - THEN' which doesn't make sense..?

Reply:
yes, it should always describe a behaviour. (Given - precondition), when - what we test, then - the behaviour we expect/do not expect.